torchtext/experimental/datasets/translation.py

```diff
@@ -0,0 +1,388 @@
+import torch
+import logging
+
+from torchtext.experimental.datasets import raw
+from torchtext.vocab import build_vocab_from_iterator
+from torchtext.data.utils import get_tokenizer
+
+
+def vocab_func(vocab):
```

Comment:
You could simply import those utils from https://github.com/pytorch/text/blob/master/torchtext/experimental/functional.py